packages/styleUtils/layout/container/components/Container.tsx

```diff
@@ -1,12 +1,19 @@
 import * as React from "react";
+import { cx } from "@emotion/css";
 import { container } from "../style";
 
 export interface ContainerProps {
   children: React.ReactNode;
+  ["data-cy"]?: string;
```

Comment:
What is this syntax called? I've not seen this before and can't seem to find anything about it in Google.

Reply:
I'm not sure what the term is for this, I had been following the previous conventions in the ui-kit for data-cy props as string values to keep it consistent. I was under the assumption it was for TS to handle the option string prop with value but I've tested now it and it works regardless. Might have been an issue that was resolved along the way with TS.

Reply:
I'm going to update it here to remove the brackets and I'll have to address the consistency separately. Thanks for pointing this out.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#computed_property_names

Reply:
oh wait, this is TS we're looking at